wrappers/IWearLogger/src/IWearLogger.cpp

```diff
@@ -25,8 +28,20 @@ constexpr double DefaultPeriod = 0.01;
 namespace wearable {
     namespace wrappers {
         struct IWearLoggerSettings;
+
+        enum class LoggerLevel
+        {
+            MATLAB = 0,
+            YARP,
+            MATLAB_YARP,
+            NONE,
+        };
     }
```

Comment:
usual comment: did you format using clang-format? I'd expect here a `// namespace wrappers`

Reply:
Done in https://github.com/robotology/wearables/pull/123/commits/2cd1dc1398e7d96cdc98ef315a80a514fd42464f